event-log/src/main/scala/io/renku/eventlog/subscriptions/SubscriptionsEndpoint.scala

```diff
@@ -23,15 +23,14 @@ import cats.effect.Effect
 import ch.datascience.graph.model.events.EventStatus
 import io.chrisdavenport.log4cats.Logger
 import io.circe.Json
-import io.renku.eventlog.subscriptions.unprocessed.SubscriptionRequestDeserializer
 import org.http4s.dsl.Http4sDsl
 
 import scala.util.control.NonFatal
 
-class SubscriptionsEndpoint[Interpretation[_]: Effect](
-    subscribers: Subscribers[Interpretation],
-    logger:      Logger[Interpretation]
-)(implicit ME:   MonadError[Interpretation, Throwable])
+class SubscriptionsEndpoint[Interpretation[_]: Effect, T <: SubscriptionCategoryPayload](
```

Comment:
Yes, hopefully we'll drop the `T` from here in the future

Reply:
Yes the registry will remove a lot of leakage :)